CMakeLists.txt

```
@@ -268,7 +268,7 @@ if (LLAMA_CUBLAS)
 
     if (NOT DEFINED CMAKE_CUDA_ARCHITECTURES)
         if (LLAMA_CUDA_DMMV_F16)
-            set(CMAKE_CUDA_ARCHITECTURES "61") # needed for f16 CUDA intrinsics
+            set(CMAKE_CUDA_ARCHITECTURES "60") # needed for f16 CUDA intrinsics
```

```suggestion
            set(CMAKE_CUDA_ARCHITECTURES "60;61") # needed for f16 CUDA intrinsics
```
As pointed out by https://github.com/ggerganov/llama.cpp/issues/2159 the minimum compute capability for `__dp4a` seems to be 6.1. So I think building PTX code for both of these architectures would be preferable.